File: Week_2/problem_11-Reversing.py
# https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/F

n = int(input())
ar = list(map(int, input().split()))

# To print the array in reversed order
for i in range(n - 1, -1, -1):
    print(ar[i], end=" ")

File: Week_2/problem_11-Reversing.c
// https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/F

#include <stdio.h>

int main()
{
    int n;
    scanf("%d", &n);
    int ar[n];

    for (int i = 0; i < n; i++)
    {
        scanf("%d", &ar[i]);
    }

    // to print the array in reversed order
    for (int j = n - 1; j >= 0; j--)
    {
        printf("%d ", ar[j]);
    }

    return 0;
}